.gitignore

```diff
@@ -36,8 +36,15 @@ yarn-debug.log*
 
 .env*
 
-# Ignore docker related files
-/vendor
+# Ignore docker related files, but don't ignore active admin's css specifications
+vendor/*
+!vendor/assets
+vendor/assets/*
+!vendor/assets/stylesheets
+vendor/assets/stylesheets/*
+!vendor/assets/stylesheets/active_admin.scss
```

Comment:
This might be a cleaner solution: https://stackoverflow.com/a/18702826. Can we try something like this out?